util/Makefile.in

```diff
@@ -95,7 +95,9 @@ APPSRC   += clixon_util_yang.c
 APPSRC   += clixon_util_xpath.c
 APPSRC   += clixon_util_path.c
 APPSRC   += clixon_util_datastore.c
+ifdef with_libxml2
```

Comment:
Actually this is too restrictive. clixon_util_regexp is needed in the regexp tests also for non-libxml2 regexps (ie posix regexps).
Preferably it should be enough with the HAVE_LIBXML2 define inside the code of clixon_util_regexp.c, so that only posix regexps are run.

Reply:
OK, I'll fix that, no problem!